src/main/java/uk/gov/ida/verifyserviceprovider/resources/GenerateAuthnRequestResource.java

```diff
@@ -24,23 +25,26 @@
 
     private final URI ssoLocation;
     private final AuthnRequestFactory authnRequestFactory;
+    private final EntityIdService entityIdHelper;
```

Comment:
Instance name is `...Helper` but class name is `...Service` - don't mind which we go for, but shouldn't they be consistent?